x-pack/plugins/fleet/server/services/fleet_usage_logger.ts

```diff
@@ -56,12 +56,25 @@ export async function startFleetUsageLogger(taskManager: TaskManagerStartContrac
   if (!isInfoLogLevelEnabled) {
     return;
   }
-  appContextService.getLogger().info(`Task ${TASK_ID} scheduled with interval 5m`);
+  const interval = isDebugLogLevelEnabled ? '5m' : '15m';
+
+  // Re-schedule the task if interval changed
```

Comment:
is this code called when the log level has changed?

Reply:
I think Kibana needs to be restarted when the log level is changed, since it's changed in `kibana.yml` or via command line flags. So this code should be called when Kibana reboots, then we check for the previously existing task.

Reply:
> I think Kibana needs to be restarted when the log level is changed, since it's changed in kibana.yml or via command line flags. So this code should be called when Kibana reboots, then we check for the previously existing task.

Yes Kibana should be restarted